pkg/kn/config/config_test.go

```diff
@@ -29,7 +29,7 @@ func TestBootstrapConfig(t *testing.T) {
 	configYaml := `
 plugins:
   directory: /tmp
-  path-lookup: true
+  
```

Comment:
not sure why this slipped in, but should not harm (as this ensure that path-lookup is the default mode)